project/program/models.py

```diff
@@ -190,11 +196,13 @@ def clean(self):
         same_time_activities = Activity.objects.filter(start=self.start)
         for other in same_time_activities:
             if Stage.from_activity(self) == Stage.from_activity(other):
-                raise ValidationError('There exists another activity that starts at the same time and stage')
+                raise ValidationError('There exists another activity that \
+                                       starts at the same time and stage')
```

Comment:
This will add whitespace from indentation, better change it to:
```python
               raise ValidationError('There exists another activity that ' \
                                     'starts at the same time and stage')
```